gravitee-apim-repository/gravitee-apim-repository-elasticsearch/src/main/java/io/gravitee/repository/elasticsearch/v4/log/LogElasticsearchRepository.java

```diff
@@ -43,4 +47,15 @@ public LogResponse<ConnectionLog> searchConnectionLog(ConnectionLogQuery query)
             .map(SearchConnectionLogResponseAdapter::adapt)
             .blockingGet();
     }
+
+    @Override
+    public LogResponse<MessageLog> searchMessageLog(MessageLogQuery query) {
+        // FIXME: Do we need this first param ? maybe rework the method to only take configuration
+        var clusters = ClusterUtils.extractClusterIndexPrefixes((ConnectionLogQuery) null, configuration);
```

Comment:
It looks like the parameter is not used in this method. I would remove it and remove the fixme 🤷🏼

Reply:
done